root/rint/TabCom_input.txt

```diff
@@ -1,4 +1,3 @@
-TObj	
```

Comment:
Why was that line useful/interesting before and is no longer of interest?

Reply:
That moved in the TTabComTests.cxx in the ROOT repository because the return results started depending on things like ROOT7 for TObjHolder and so on. That is why I decided to sink that code there to be able to check better for configuration.